cvat/utils/throttle.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from django.core.cache import caches
+from rest_framework.throttling import AnonRateThrottle
+
+
+class SharedCacheThrottle(AnonRateThrottle):
```

Comment:
Is this supposed to inherit from `AnonRateThrottle`? I assume the point of this class is to be a mixin for different types of throttles.

Reply:
Oh, exactly, my bad